### PR TITLE
Increase tests' and examples' compatibility with OMEdit

### PR DESCRIPTION
**Pull request by marcelofcastro**
This PR is supposed to fix:
- Initial condition information which prevented models from simulating in OpenModelica
- Warning related to Fault Event (by replacing 0 to C.eps in some examples and tests)

Therefore, issue #160  is addressed in this PR (although the branch was mistakenly called ...245-fixes).

### CODE REVIEW

**Comment by dietmarw** [comment]
Any chance to isolate the softPMU part into a separate PR. I still see quite a bit of an issue there and don't want it to be in the way of the OM fixes.

**Comment by marcelofcastro** [comment]
I think we can discuss here the issues you see with the SoftPMU thing

File: OpenIPSL/Examples/SevenBus/Network.mo [hunk 1]
```diff
@@ -161,38 +158,31 @@ model Network "Seven-bus test system model, ready for simulation"
...
-    VB1=380000,
-    VNOM2=380000,
-    VB2=380000,
-    ANG1=0,
     t1=PF_results.trafos.t1_1_4,
     t2=PF_results.trafos.t2_1_4)
     annotation (Placement(transformation(extent={{-12,-40},{8,-24}})));
+
   SevenBus.Generators.G2 GEN2(
+    M_b(displayUnit = "V.A") =1710000000,
```
Comment (dietmarw) [inline OpenIPSL/Examples/SevenBus/Network.mo:171]: This should better be `"MVA"`

File: OpenIPSL/Examples/SevenBus/Network.mo [hunk 2]
```diff
@@ -161,38 +158,31 @@ model Network "Seven-bus test system model, ready for simulation"
...
-    v_0=PF_results.voltages.V61,
     angle_0=PF_results.voltages.A61,
-    P_0=PF_results.machines.P61_1,
-    Q_0=PF_results.machines.Q61_1)
+    v_0=PF_results.voltages.V61)
     annotation (Placement(transformation(extent={{280,-142},{260,-122}})));
+
   SevenBus.Generators.G3 GEN3(
-    M_b=1211000000,
+    M_b(displayUnit = "V.A") =1211000000,
```
Comment (dietmarw) [inline OpenIPSL/Examples/SevenBus/Network.mo:180]: MVA

File: OpenIPSL/Examples/SevenBus/Network.mo [hunk 3]
```diff
@@ -204,63 +194,59 @@ model Network "Seven-bus test system model, ready for simulation"
...
+        transformation(
+        origin={-248,-132},
+        extent={{-10,-10},{10,10}},
+        rotation=0)));
+  Electrical.Buses.Bus internal_bus_gen3 annotation (Placement(visible=true,
+        transformation(
+        origin={0,128},
+        extent={{-10,-10},{10,10}},
+        rotation=90)));
+  Electrical.Buses.Bus internal_bus_gen2(S_b(displayUnit = "V.A") = 1710000000,V_b = 20000,angle_0=PF_results.voltages.A61, v_0=
```
Comment (dietmarw) [inline OpenIPSL/Examples/SevenBus/Network.mo:244]: MVA

File: OpenIPSL/Tests/BaseClasses/LoadTestBase.mo [hunk 4]
```diff
@@ -64,42 +55,75 @@ partial model LoadTestBase "Base class for the different load cases."
         origin={-84,-16},
         extent={{-4,-4},{4,4}})));
   OpenIPSL.Electrical.Machines.PSAT.Order3 order3_Inputs_Outputs1(
-    P_0=800989.8784778,
+    D=0,
+    M=10,P_0=800989.8784778,
     Q_0=570163.38872796,
-    Sn=20000000,
+    Sn(displayUnit = "V.A")= 20000000,
```
Comment (dietmarw) [inline OpenIPSL/Tests/BaseClasses/LoadTestBase.mo:61]: MVA

File: OpenIPSL/Tests/BaseClasses/TGTestBase.mo [hunk 5]
```diff
@@ -14,85 +14,108 @@ partial model TGTestBase "Base class for tests involving turbine governors."
...
     D=0,
-    xd=1.9,
-    T1d0=8,
-    xq=1.7,
+    M=10,
     P_0=160410,
     Q_0=120120,
-    Sn=20000000,
-    v_0=1,
+    Sn(displayUnit = "V.A") =20000000,
```
Comment (dietmarw) [inline OpenIPSL/Tests/BaseClasses/TGTestBase.mo:69]: MVA

**Review by dietmarw — CHANGES_REQUESTED** [review]
Please remove all the `"V.A"` and `"V"` displayUnit modifiers. Those are already set correctly in the types and this override does not make sens and makes the numbers even harder to read.

**Review by dietmarw — APPROVED** [review]
Looks good now.

**Pull request merged**